api/app/signals/settings/base.py

```diff
@@ -217,7 +216,7 @@
     }
 """
 
-DATAPUNT_AUTHZ = {
+SIGNALS_AUTHZ = {
```

Comment:
Add 'USER_ID_FIELD': `os.getenv('USER_ID_FIELD', 'email')` here so we can configure this by environment variables.